src/schemas/json/winget-pkgs.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "properties": {
+        "Id": {
+            "type": "string",
+            "description": "publisher.package format"
+        },
+        "Publisher": {
+            "type": "string",
+            "description": "the name of the publisher"
+        },
+        "Name": {
+            "type": "string",
+            "description": "the name of the application"
+        },
+        "Version": {
+            "type": "string",
+            "description": "version numbering format"
+        },
+        "License": {
+            "type": "string",
+            "description": "the open source license or copyright"
+        },
+        "InstallerType": {
+            "type": "string",
+            "description": "enumeration of supported installer types (exe, msi, msix)"
+        },
```

Comment:
This should be under the properties in the Installers section with Arch, URL and Sha 256 since there is where the tool script provided in the winget-pkg repo generates it.